codice/ddf
# [2.26.x] Skip all itests for OWASP scan and only run aggregate goal for CI

#### What does this PR do?
This is a backport of https://github.com/codice/ddf/pull/6347

It speeds up the OWASP scan step of the Jenkins build and reduces noise in the reports by ignoring docs and itest dependencies.
 
#### Who is reviewing it? 


#### Select relevant component teams: 


#### Ask 2 committers to review/merge the PR and tag them here.


#### How should this be tested?

Run CI and verify that:
* OWASP report is available
* Report contains no CVEs for docs or itest dependencies

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #____

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
